src/main/scala/ch/epfl/bluebrain/nexus/kg/resolve/InProjectResolution.scala

```diff
@@ -0,0 +1,26 @@
+package ch.epfl.bluebrain.nexus.kg.resolve
+
+import cats.Functor
+import cats.syntax.functor._
+import ch.epfl.bluebrain.nexus.kg.resources.Ref._
+import ch.epfl.bluebrain.nexus.kg.resources._
+
+/**
+  * Simplest implementation that handles the resolution process of references to resources
+  * within a given project.
+  *
+  * @param project the resolution scope
+  * @param repo    the implicitly available resource repository
+  * @tparam F      the resolution effect type
+  */
+class InProjectResolution[F[_]: Functor](project: ProjectRef)(implicit repo: Repo[F]) extends Resolution[F] {
```

Comment:
could you also add a companion object to construct the resolution instance without the new operator?
`InProjectResolution(project)`